Libplanet.Net/Consensus/ConsensusContext.cs

```diff
@@ -327,7 +327,24 @@ private void OnTipChanged(object? sender, (Block<T> OldTip, Block<T> NewTip) e)
             Task.Run(
                 async () =>
                 {
-                    await Task.Delay(_newHeightDelay, _newHeightCts.Token);
+                    var startTime = _contexts.ContainsKey(e.NewTip.Index)
+                        ? _contexts[Height].StartTime
+                        : DateTimeOffset.MinValue;
```

Comment:
Highly unlikely, but this could cause an unexpected exception due to (lack of) concurrency.
Using `_contexts.TryGetValue()` would be more apt here.

Reply:
Or we could just use context `lock` like everywhere else. 😗

Reply:
Seems better to use `TryGetValue` here, to minimize `lock`. Thanks.